### DISCUSSION TITLE
_layout folder is gone?

### DISCUSSION THREAD

**Discussion by erindarrow** [community/community#47081] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi! I'm trying to change some content on m[y page](https://www.erindarrow.com) and noticed that the `_layouts/` folder is gone. I must have done something wrong somewhere and I'm wondering how to rectify this. I also notice I now have a _site folder which I don't think I had earlier in working on the site.

Can anyone offer some guidance or point me in the right direction? I'm feeling a little confused. thanks!

If helpful, the relevant repository is here: https://github.com/erindarrow/erindarrow.github.io
using a remote theme from: https://github.com/daviddarnes/alembic

**Comment by airtower-luna** [comment]
Where do you expect the `_layouts/` folder to be? Looking at [the history of your repository](https://github.com/erindarrow/erindarrow.github.io/commits/main) the `_site/` folder is there from the [oldest commit](https://github.com/erindarrow/erindarrow.github.io/commit/4700f344124a4c8b4a3c5da41fcc109254c2a8c8), and there's no `_layout/`.

**Comment by erindarrow** [comment]
Yes, I saw that too when I looked through it... I expected it to be in the main repository.

I think I may be getting confused with another theme I had tried using previously. For that one, I forked the repo and made changes and there was a _layout folder whereas it seems with using the remote theme, the _layouts doesn't "live" (for lack of better word) in my repository. Does that sound accurate or am I way off about this?

**Comment by airtower-luna** [comment, marked as answer]
That seems about right, you can see the theme being downloaded in the build log: https://github.com/erindarrow/erindarrow.github.io/actions/runs/4152603752/jobs/7183676352#step:4:48

**Reply by erindarrow** [reply]
Ahhh ok. thank you! So I just confused myself is all. I appreciate the assistance.
